build/stubs/PhpParserName.stub

```diff
@@ -0,0 +1,25 @@
+<?php declare(strict_types=1);
+
+namespace PhpParser\Node;
+
+use PhpParser\NodeAbstract;
+
+class Name extends NodeAbstract
```

Comment:
we should be able to remove this stub after the next php-parser release because of https://github.com/nikic/PHP-Parser/pull/993